src/Goerli-DssSpell.sol

```diff
@@ -89,58 +54,91 @@ contract DssSpellAction is DssAction, DssSpellCollateralAction {
-        DssExecLib.setChangelogAddress("STARKNET_TELEPORT_FEE", LINEAR_FEE_STA);
-
-        DssExecLib.setChangelogVersion("1.14.4");
-
-        // ------------------ MOMC Parameter Changes -----------------
-        // https://vote.makerdao.com/polling/QmahDuNx#poll-detail
-
-        // Increase the MANA-A Stability Fee from 4.5% to 7.5%
-        DssExecLib.setIlkStabilityFee("MANA-A", SEVEN_PT_FIVE_PERCENT_RATE, true);
+        // Init the RwaLiquidationOracle to reset the doc
+        RwaLiquidationLike(MIP21_LIQUIDATION_ORACLE).init(
+            ilk, // ilk to update
+            0,   // price ignored if init() has already been called
+            doc, // new legal document
+            tau  // old tau value
+        );
+    }
 
-        // Decrease the MANA-A line from 17 million DAI to 10 million DAI
-        DssExecLib.setIlkAutoLineDebtCeiling("MANA-A", 10 * MILLION);
+    function actions() public override {
+        // -------------------- Update RWA007, RWA008, RWA009 Legal Documents ---------------------
+        // https://forum.makerdao.com/t/nov-16-2022-executive-contents/18747
+        //
+        // Monetalis (RWA007-A) legal update doc
+        _updateDoc("RWA007-A", "QmejL1CKKN5vCwp9QD1gebnnAM2MJSt9XbF64uy4ptkJtR");
+        // SG Forge OFH (RWA008-A) legal update doc
+        _updateDoc("RWA008-A", "QmZ4heYjptvj3ovafADJpXYMFXMyY3yQjkTXpvjFPnAKcy");
+        // HVB (RWA009-A) legal update doc
+        _updateDoc("RWA009-A", "QmeRrbDF8MVPQfNe83gWf2qV48jApVigm1WyjEtDXCZ5rT");
+
+        // RWA007-A autoline changes:
+        // - bump oralce price to 500m
+        // - increase DC to 500m
+        // - increase autoline `gap` to 100m
+        //
+        // https://vote.makerdao.com/polling/QmSfMtTM#poll-detail
+        DssExecLib.setIlkAutoLineParameters(
+            "RWA007-A",
+            500 * MILLION,
+            100 * MILLION,
+            1 weeks
+        );
+        RwaLiquidationLike(MIP21_LIQUIDATION_ORACLE).bump(
+            "RWA007-A",
+             500 * MILLION * WAD
+        );
+        Pricing(DssExecLib.spotter()).poke("RWA007-A");
+
+        // RETH-A autoline changes:
+        // - line 5m
+        // - gap 3m
+        // - ttl 8h
+        //
+        //  https://vote.makerdao.com/polling/QmfMswF2#poll-detail
+        DssExecLib.setIlkAutoLineParameters(
+            "RETH-A",
+            5 * MILLION,
+            3 * MILLION,
+            8 hours
+        );
+
+        // MANA-A param changes:
+        // - decrease line to 3m
+        // - increase SF to 50%
+        // - increase liquidation penalty to 30% from current 13%
+        //
+        // https://forum.makerdao.com/t/mana-a-intermediate-parameter-change-proposal/18727
+        DssExecLib.setIlkAutoLineDebtCeiling("MANA-A", 3 * MILLION);
+        DssExecLib.setIlkStabilityFee("MANA-A", FIFTY_PCT_RATE, true);
+        DssExecLib.setIlkLiquidationPenalty("MANA-A", 3000); // 30%;
+
```

Comment:
fyi: following this [forum post](https://forum.makerdao.com/t/urgent-signal-request-urgent-recommended-collateral-parameter-changes/18764), these changes might be moved to #138

Reply:
done